packages/cli/core/src/commands/auth/__snapshots__/login.spec.ts.snap

```diff
@@ -0,0 +1,32 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`auth:login succeed when no organization flag is passed, and uses the first available org instead 1`] = `
+"
+    Successfully logged in!
+    Close your browser to continue.
+
+    You are currently logged in:
+    Organization: the_first_org_available
+    Region: us
+    Environment: prod
+    
+To changes organizations either:
+  * Run coveo auth:login --help to see the available options to log in to a different organization, region, or environment.
```

Comment:
I don't really understand here: the `either` seems to imply that there are two options, but I only see one?

I'm guessing that the content of this file gets generated from the one below? I will suggest something in that file, but let me know if I'm wrong and I'll suggest something here too 👍